README.md

```diff
@@ -117,7 +117,9 @@ Start by [adding widgets](https://community.algolia.com/instantsearch.js/v2/widg
 
 ## Browser support
 
-We support the last **two versions of major browsers**. To support [IE11](https://en.wikipedia.org/wiki/Internet_Explorer_11), we recommend using [polyfill.io](https://polyfill.io).
+We support the last **two versions of major browsers** (Chrome, Edge, Firefox, Safari).
```

Comment:
"last" should also probably be `strong` → `**last two versions of major browsers**`.